container/mesos/factory.go

```diff
@@ -107,16 +107,11 @@ func (self *mesosFactory) CanHandleAndAccept(name string) (handle bool, accept b
 	// Check if the container is known to mesos and it is active.
 	id := ContainerNameToMesosId(name)
 
-	c, err := self.client.ContainerInfo(id)
+	_, err = self.client.ContainerInfo(id)
```

Comment:
do we still need to make this call?

Reply:
Yes as part of factory, we need to see if the container info object  can be created with the given id.